api.json

```diff
@@ -1406,6 +1453,47 @@
              "$ref":"#/components/schemas/Amount"
             }
           },
+         "metadata": {
+           "description":"Account-based blockchains that utilize a nonce or sequence number should include that number in the metadata. This number could be unique to the identifier or global across the account address.",
+           "type":"object",
+           "example": {
+             "sequence_number": 23
+            }
+          }
+        }
+      },
+     "AccountCoinsRequest": {
+       "description":"AccountCoinsRequest is utilized to make a request on the /account/coins endpoint.",
+       "type":"object",
+       "required": [
+         "network_identifier",
+         "account_identifier",
+         "include_mempool"
+        ],
+       "properties": {
+         "network_identifier": {
+           "$ref":"#/components/schemas/NetworkIdentifier"
+          },
+         "account_identifier": {
+           "$ref":"#/components/schemas/AccountIdentifier"
+          },
+         "include_mempool": {
+           "type":"boolean",
+           "description":"Include state from the mempool when looking up an account's unspent coins. Note, using this functionality breaks any guarantee of idempotency."
+          }
+        }
+      },
+     "AccountCoinsResponse": {
+       "description":"AccountCoinsResponse is returned on the /account/coins endpoint and includes all unspent Coins owned by an AccountIdentifier.",
```

Comment:
`nit` might want to saying something like `owned by an AccountIdentifier at a BlockIdentifier`?

Reply:
I left this purposely open-ended because it may include coins from the mempool (not at a BlockIdentifier strictly).